webapp/src/components/link_embed_preview/link_embed_preview.jsx

```diff
@@ -0,0 +1,184 @@
+import {GitMergeIcon, GitPullRequestIcon, IssueClosedIcon, IssueOpenedIcon, SkipIcon} from '@primer/octicons-react';
+import PropTypes from 'prop-types';
+import React, {useEffect, useState} from 'react';
+import ReactMarkdown from 'react-markdown';
+import './embed_preview.css';
+
+import Client from 'client';
+import {getLabelFontColor} from '../../utils/styles';
+import {isUrlCanPreview} from '../../utils/github_utils';
+
+const maxTicketDescriptionLength = 160;
+
+export const LinkEmbedPreview = ({embed: {url}, connected}) => {
+    const [data, setData] = useState(null);
+    useEffect(() => {
+        const initData = async () => {
+            if (isUrlCanPreview(url)) {
+                const [owner, repo, type, number] = url.split('github.com/')[1].split('/');
```

Comment:
Can this cause issues if someone crafts a URL with `github.com/` in the subpath?

Reply:
Yes, I checked it occurs on tooltip too so I took action for it